starport/templates/app/stargate/vue/README.md.plush

```diff
@@ -1,8 +1,8 @@
 # <%= AppName %> UI
 
-This is the [vuejs](https://vuejs.org/) user inteface to <%= AppName %>.
+This is the [vuejs](https://vuejs.org/) user interface to myblockchain.
```

Comment:
```suggestion
This is the [vuejs](https://vuejs.org/) user interface to <%= AppName %>.
```

We should keep the placeholder, because it will replaced with a custom name.